### DISCUSSION TITLE
Need help understanding how the watch-only internal wallet works to send funds

### DISCUSSION THREAD

**Discussion by v-wan** [btcpayserver/btcpayserver#5929] [General]
Due to concerns of anyone being able to generate more than 20 invoices to cause a gap limit problem, I've decided to use the internal watch-only wallet for managing funds. That's great, but I don't understand what private key I need to input to sign the transaction when I need to send funds to my main wallet, as the only thing I got during the wallet setup was the seed phrase.

**Comment by v-wan** [comment]
Here are a few screenshots that I am confused about:
![image](https://github.com/btcpayserver/btcpayserver/assets/134129286/7ba772a3-c81e-4ff0-8566-65ef89b91ce1)
I decide to use the internal wallet, so I keep reading...

![image](https://github.com/btcpayserver/btcpayserver/assets/134129286/06496bc6-ae57-4eb3-a67c-21239b426456)
I click into the link and head into the page that describes the internal wallet

![image](https://github.com/btcpayserver/btcpayserver/assets/134129286/b2147db2-b0f7-4cf7-b625-c245b1ee3c4e)
Cool, so now I see that the internal wallet is able to send funds

![image](https://github.com/btcpayserver/btcpayserver/assets/134129286/e8a7f28b-4464-4b9a-bf91-5e5b2c0bc599)
Looking at the screenshot of the Send page, I can see the destination address field where I will input the address of my main wallet, so that looks good, but to sign the transaction I need to create a new external wallet in BTCPayServer, when I just created an internal watch-only wallet?

![image](https://github.com/btcpayserver/btcpayserver/assets/134129286/5b3db668-598e-4e7c-bc90-01376a0dd16c)
Honestly confused

![image](https://github.com/btcpayserver/btcpayserver/assets/134129286/b5585307-fc9c-4a07-91f8-cde059bc4af8)

**Reply by pavlenex** [reply]
To sign enter the backup words.

**Reply by v-wan** [reply]
Ah cool, thanks!
